timestamp-impl/src/main/java/com/palantir/timestamp/TimestampAllocationFailures.java

```diff
@@ -32,7 +34,7 @@
     private final Logger log;
 
     private volatile Throwable previousAllocationFailure;
-    private volatile boolean encounteredMultipleRunningTimestamps = false;
+    @Contended private volatile boolean encounteredMultipleRunningTimestamps = false;
```

Comment:
heads up - [00:00:20] /home/ubuntu/atlasdb-internal-testing/timestamp-impl/src/main/java/com/palantir/timestamp/TimestampAllocationFailures.java:26: warning: Contended is internal proprietary API and may be removed in a future release